index.js

```diff
@@ -16,6 +16,10 @@ function Attempt(instance, end) {
   var command = [];
   command.push('/usr/bin/sudo');
   command.push('-n');
+  // Use localhost as sudo may stderr 'sudo: unable to resolve host <hostname>'
+  // if the hostname is not in /etc/hosts while still executing the command.
+  command.push('-h');
+  command.push('localhost');
```

Comment:
Should this maybe use `127.0.0.1` instead, as `localhost` doesn't necessarily resolve to `127.0.0.1`?

Reply:
Good point.

Reply:
@jorangreef the PR is already updated to use `127.0.0.1`